elide-integration-tests/src/test/java/com/yahoo/elide/tests/AnyPolymorphismIT.java

```diff
@@ -216,41 +195,57 @@ public void testAnySubpaths() {
                 .given()
                 .contentType(JSONAPI_CONTENT_TYPE)
                 .accept(JSONAPI_CONTENT_TYPE)
-                .body(jsonParser.getJson(tractorAsPropertyFile))
-                .post(propertyPath)
+                .body(jsonParser.getJson("/AnyPolymorphismIT/AddTractorProperty.json"))
+                .post("/property")
                 .then()
                 .assertThat()
                 .statusCode(HttpStatus.SC_CREATED)
                 .extract()
-                .path(idPath);
+                .path("data.id");
 
-        RestAssured
+        Response response = RestAssured
                 .given()
                 .contentType(JSONAPI_CONTENT_TYPE)
                 .accept(JSONAPI_CONTENT_TYPE)
-                .get(propertyPath + "?page[totals]")
+                .get("/property?page[totals]")
                 .then()
                 .assertThat()
                 .statusCode(HttpStatus.SC_OK)
-                .body("meta.page.totalRecords", greaterThan(0));
+                .extract()
+                .response();
+
+        Integer collectionSize = response.path("data.size()");
+        Integer totalSize = response.path("meta.page.totalRecords");
+        Assert.assertEquals(collectionSize, totalSize);
 
         RestAssured
                 .given()
                 .contentType(JSONAPI_CONTENT_TYPE)
                 .accept(JSONAPI_CONTENT_TYPE)
-                .get(propertyPath + sep + id + "/myStuff")
+                .get("/property/" + id + "/myStuff")
                 .then()
                 .assertThat()
                 .statusCode(HttpStatus.SC_OK)
-                .body("data.attributes.horsepower", equalTo(horsepower),
-                        idPath, equalTo(one));
+                .body("data.attributes.horsepower", equalTo(102),
+                        "data.id", equalTo("1"));
+
+        //show that you can't get a relation by the interface type
+        RestAssured
+                .given()
+                .contentType(JSONAPI_CONTENT_TYPE)
+                .accept(JSONAPI_CONTENT_TYPE)
+                .get("/property/" + id + "/device")
+                .then()
+                .assertThat()
+                .statusCode(HttpStatus.SC_NOT_FOUND)
+                .body("errors[0]", containsString("Unknown collection"));
 
         //single entity so no page appropriate stuff
         RestAssured
                 .given()
                 .contentType(JSONAPI_CONTENT_TYPE)
                 .accept(JSONAPI_CONTENT_TYPE)
-                .get(propertyPath + sep + id + "/myStuff?page[totals]")
+                .get("/property/" + id + "/myStuff?page[totals]")
```

Comment:
I still don't think "meta" should be null below.  This looks like a bug.

Reply:
this means that meta isn't a valid jsonpath. (meta doesn't exist in the response), not that meta: null